internal/lookout/repository/schema/014_add_reason.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE job_run ADD COLUMN preempted timestamp null;
```

Comment:
nit: The name of this sql file doesn't really match the code?  Should this be 014_add_preempted.sql?

Reply:
good nit, will change